src/components/shared/pickers/DatePickerCTA.tsx

```diff
@@ -24,33 +24,19 @@ function DatePickerCTA(props: PickerProps) {
             {...props}
         >
             <StaticDatePicker
-                ignoreInvalidInputs
-                disableMaskedInput
                 displayStaticWrapperAs="desktop"
                 openTo="day"
                 disabled={!enabled}
-                value={value}
-                onChange={(
-                    onChangeValue: any,
-                    keyboardInput?: string | undefined
-                ) => {
+                defaultValue={parseISO(value)}
+                onChange={(onChangeValue: any) => {
```

Comment:
The keyboardInput was really just to be super super safe and I never saw it actually get hit in my testing ever. Since that is no longer getting passed in I am removing it.